userspace/libsinsp/test/CMakeLists.txt

```diff
@@ -58,7 +58,9 @@ if((NOT ${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "s390x") AND ${SCAP_FILES_SUITE_
 	file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/scap_files")
 	# Add here the name for new scap-files
 	set(SCAP_FILE_NAMES "kexec_arm64.scap" "kexec_x86.scap" "sample.scap")
-	set(SCAP_FILE_DOWNLOAD_PREFIX "https://download.falco.org/fixtures/libs/scap_files")
+	set(SCAP_FILE_DOWNLOAD_PREFIX
```

Comment:
To workaround some scarf-related issues, just skip using it while downloading text fixtures.